app/components/network_indicator/network_indicator.js

```diff
@@ -112,15 +112,9 @@ export default class NetworkIndicator extends PureComponent {
     componentWillUnmount() {
         this.mounted = false;
 
-        const {closeWebSocket, stopPeriodicStatusUpdates} = this.props.actions;
-
         this.networkListener.removeEventListener();
-
         AppState.removeEventListener('change', this.handleAppStateChange);
 
-        closeWebSocket();
-        stopPeriodicStatusUpdates();
-
         clearTimeout(this.connectionRetryTimeout);
```

Comment:
Unrelated, but perhaps set this to `null` afterwards, as per typical timeout cleanup semantics. Only helps with racey shutdowns, but my mind has been there of late ;)

Reply:
@lieut-data done